README.md

```diff
@@ -1,11 +1,23 @@
-![Margarita](assets/logo.png)
+<div align="center">
+<img src="./assets/logo.png" alt="Margarita" title="Margarita" width="300"/>
+</div>
 
 # Margarita
 
 > Showcasing the power of [Tequila](https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/)
 
 This project is a living example of a client app you could start building with Tequila.
 
+- 🕸️ Web version: https://kiwicom-margarita.netlify.com
+- 🎮 GraphQL Playground https://p2kwd3i3a8.execute-api.eu-central-1.amazonaws.com/staging/graphql
+- 🚀 GraphQL Voyager: https://margarita-graphql-voyager.now.sh/
```

Comment:
I am not sure if we should add this voyager to documentation because it is not reflected current GraphQL schema it's deployed from this branch `https://github.com/kiwicom/margarita/tree/voyager/apps/voyager`, it should be merged first otherwise, our voyager is outdated.

Reply:
@michalsanger Shall we create a PR from your branch?

Reply:
The Voyager now loads schema from `https://p2kwd3i3a8.execute-api.eu-central-1.amazonaws.com/staging/graphql` so it should be up to date.

The PR can't be done now, it's build on CRA that has deps conflicts because of workspaces. I even tried eject and some fixes, gave up after 2 hours as it's not urgent to have it done :-(

Reply:
So if it this updated we can merge this PR, thanks for the information.